### ISSUE TITLE
Sonarr 3.0.4.995 fails to import from qbit 4.3.0.1 because of a forward-slash in the path

### ISSUE THREAD

**Issue by ddevenport** [Sonarr/Sonarr#4094]
**Describe the bug**
Sonarr fails to import files from qbit version 4.3.0.1 because of a forward slash in the path. Looks relevant to this: https://github.com/Sonarr/Sonarr/pull/3346#issuecomment-727063745

**Screenshots**
N/A
**Logs**
```
[v3.0.4.995] System.Exception: Cannot combine OsPaths of different platforms ('C:\Users\david\Downloads\qBittorrent' + 'The.Blacklist.S08E01.720p.HDTV.x264-SYNCOPY[TGx]/')
   at NzbDrone.Common.Disk.OsPath.op_Addition(OsPath left, OsPath right) in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Common\Disk\OsPath.cs:line 288
   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.GetImportItem(DownloadClientItem item, DownloadClientItem previousImportAttempt) in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\Clients\QBittorrent\QBittorrent.cs:line 256
   at NzbDrone.Core.Download.ProvideImportItemService.ProvideImportItem(DownloadClientItem item, DownloadClientItem previousImportAttempt) in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\PrepareImportService.cs:line 21
   at NzbDrone.Core.Download.CompletedDownloadService.Check(TrackedDownload trackedDownload) in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\CompletedDownloadService.cs:line 64
   at NzbDrone.Core.Download.TrackedDownloads.DownloadMonitoringService.ProcessClientItem(IDownloadClient downloadClient, DownloadClientItem downloadItem) in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Download\TrackedDownloads\DownloadMonitoringService.cs:line 123
```

**System Information**

 - Sonarr Version: 3.0.4.995
 - Operating System: Windows 10
 - .net Framework (Windows) or mono (macOS/Linux) Version: .net 4.8.0

**UI Bugs:**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 86.0.4240.193

**Additional context**
https://github.com/Sonarr/Sonarr/blob/2d94857369ede867848da1f35658a9790fc51187/src/NzbDrone.Common/Disk/OsPath.cs#L288

The actual directory is `C:\Users\david\Downloads\qBittorrent\The.Blacklist.S08E01.720p.HDTV.x264-SYNCOPY[TGx]` so it looks like it's somehow placing the / in the path. That is not in the torrent name displayed in qbit and doesn't show up when I copy the name from the context menu.

**Comment by bakerboy448** [comment]
Fixed in 3.0.4.999
https://github.com/Sonarr/Sonarr/commit/3ff848b0190212bc0470f0e182e602b076ecdd80

**Comment by ddevenport** [comment]
I jumped the gun to update after that referenced PR that I didn't see this other release today. Thanks!
